RELEASE_NOTES.md

```diff
@@ -1,5 +1,214 @@
 # Release Notes
 
+## 4.8.0
```

Comment:
I don't see a note about the HereDoc notation in Containerfiles.  That was dragged in by Buildah v1.33 and should be noted somewhere.

buildah: add heredoc support for RUN, COPY and ADD by @flouthoc in https://github.com/containers/buildah/pull/5092